src/pymor/models/basic.py

```diff
@@ -84,6 +86,48 @@ def __str__(self):
     def _compute_solution(self, mu=None, **kwargs):
         return self.operator.apply_inverse(self.rhs.as_range_array(mu), mu=mu)
 
+    def _compute_solution_d_mu_single_direction(self, parameter, index, solution, mu):
+        lhs_d_mu = self.operator.d_mu(parameter, index).apply(solution, mu=mu)
+        rhs_d_mu = self.rhs.d_mu(parameter, index).as_range_array(mu)
+        rhs_operator = rhs_d_mu - lhs_d_mu
+        return self.operator.apply_inverse(rhs_operator, mu=mu)
```

Comment:
```suggestion
        rhs = rhs_d_mu - lhs_d_mu
        return self.operator.jacobian(solution, mu=mu).apply_inverse(rhs)
```

Otherwise, the formula is false for nonlinear operators, right?